src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -162,28 +198,50 @@ class ActivityFeed extends React.Component<Props, State> {
         this.setState({ mentionSelectorContacts: getMentionWithQuery(searchStr) });
     };
 
+    componentDidMount(): void {
+        const { comments, tasks, versions } = this.props;
+        this.sortFeedItems(comments, tasks, versions);
+    }
+
+    componentWillReceiveProps(nextProps: any): void {
```

Comment:
Prevent the extra sorts as we discussed on hangouts

Reply:
done

Reply:
Am not sure about this part. I get how we want to have a separate state for the pending items and sorted, but i dont  agree that adding or editing should only affect the state in ActivtyFeed and not ContentSidebar as well

Reply:
We no longer have `pendingItems` as a state property. Are you suggesting we create/edit/delete all comments in ContentSidebar, and not in ActivityFeed?

Reply:
Lets discuss this when we go over your investigation